src/hooks/useDocumentDriveServer.ts

```diff
@@ -341,6 +347,20 @@ export function useDocumentDriveServer(
         );
     }
 
+    async function getSyncStatus(
+        driveId: string,
+        type: DriveType,
+    ): Promise<SyncStatus | undefined> {
+        if (getIsLocalDrive(type)) return;
+        try {
+            server.getSyncStatus(driveId);
```

Comment:
If `server.getSyncStatus` returns a status it is not being returned right? Unless I'm misunderstanding the code.

Reply:
woops yeah I must have somehow lost my return statement when I was refactoring.

fixed and pushed now.